website/docs/help-and-support/troubleshooting-guide/query-errors.md

```diff
@@ -5,6 +5,16 @@ description: >-
 # Query Errors
 The section illustrates common query errors and how to resolve them on Appsmith.
 
+## Find error messages from the datasource
+
+When a query fails, Appsmith commonly shows a notification stating something like: `UncaughtPromiseRejection: <query_name> failed to execute.`
+
+![](/img/as_error.png)
+
+To see the error message returned by the datasource in response to your query, look for the failed query in the [debug console](/help-and-support/troubleshooting-guide/#using-the-linter-and-debugger), or bind `<query_name>.data` to a [Text widget](/reference/widgets/text/) to see what it says.
```

Comment:
As you are demonstrating it nicely in the video, let's not explicitly ask users to bind it to a text widget to see the error message. Because they could be also trying to display the error using JSObject. Just let them know that they can use `<query_name>.data` to see the actual error message returned by the datasource.

Reply:
Updated